src/lib/types/CommonInputProps.ts

```diff
@@ -10,6 +10,7 @@ interface CommonInputProps<T extends FieldValues> {
   helpText?: ReactNode;
   disabled?: boolean;
   labelToolTip?: string;
+  markAllOnFocus?: boolean;
```

Comment:
Since this is added to the `CommonInputProps` all Inputs should support it. 

So either please implement it to the typeaheads as well or use the `Omit` to remove this property for those typeheads. 

Thanks :-)

Reply:
i must have missed the typeaheads many thanks, my target was it to add it for all, so typeahead + tests are added.

Reply:
nice, thx!